tasks/main-clevis.yml

```diff
@@ -4,13 +4,14 @@
     name: "{{ __nbde_client_packages }}"
     state: present
 
+- name: Get services
+  service_facts:
+
 - name: Enable clevis askpass unit
   service:
     name: clevis-luks-askpass.path
     enabled: yes
-  when: ansible_distribution != "RedHat" or
-    (not ansible_distribution_version is version("8.2", "==") and
-     not ansible_distribution_version is version("8.3", "=="))
+  when: ansible_facts.services['clevis-luks-askpass.service'] is defined
```

Comment:
@sergio-correia is this service created when you install the `clevis` RPM packages? If so, then this looks correct.

Reply:
According to `dnf` the service is provided by `clevis-systemd`.

```
[root@mt-test01 ~]# dnf whatprovides "/usr/*/clevis-luks-askpass*"
Updating Subscription Management repositories.
Last metadata expiration check: 0:09:53 ago on Fri 30 Sep 2022 08:16:08 PM UTC.
clevis-systemd-11-2.el8.x86_64 : systemd integration for clevis
Repo        : rhel-8-for-x86_64-appstream-rpms
Matched from:
Filename    : /usr/lib/systemd/system/clevis-luks-askpass.path
Filename    : /usr/lib/systemd/system/clevis-luks-askpass.service
Filename    : /usr/libexec/clevis-luks-askpass

clevis-systemd-11-9.el8.x86_64 : systemd integration for clevis
Repo        : rhel-8-for-x86_64-appstream-rpms
Matched from:
Filename    : /usr/lib/systemd/system/clevis-luks-askpass@.path
Filename    : /usr/lib/systemd/system/clevis-luks-askpass@.service
Filename    : /usr/libexec/clevis-luks-askpass

...

clevis-systemd-15-8.el8.x86_64 : systemd integration for clevis
Repo        : rhel-8-for-x86_64-appstream-rpms
Matched from:
Filename    : /usr/lib/systemd/system/clevis-luks-askpass.path
Filename    : /usr/lib/systemd/system/clevis-luks-askpass.service
Filename    : /usr/libexec/clevis-luks-askpass
```

Reply:
> @sergio-correia is this service created when you install the `clevis` RPM packages? If so, then this looks correct.

Yeah, it's installed by the `clevis-systemd` package. And I agree it looks correct.